crowdin.yml

```diff
@@ -1,5 +1,12 @@
-files:
-  - source: /iNaturalist/src/main/res/values/strings.xml
-    translation: /iNaturalist/src/main/res/values-%android_code%/strings.xml
-  - source: iNaturalist/src/main/res/values/strings.xml
-    translation: /iNaturalist/src/main/res/values-%two_letters_code%/strings.xml
+{
+  "files": [
+    {
+      "source": "/iNaturalist/src/main/res/values/strings.xml",
+      "translation": "/iNaturalist/src/main/res/values-%two_letters_code%/strings.xml",
+      "dest": "/Android/strings.xml"
+    }
+  ],
+  "project_id_env": "CROWDIN_PROJECT_ID",
+  "api_token_env": "CROWDIN_PERSONAL_TOKEN",
+  "preserve_hierarchy": true
+}
```

Comment:
I realize that JSON is a subset of YAML, but why the switch?

Reply:
Not mandatory, it was just because while I was developing it, all the examples in the documents were JSON formatted

Reply:
Will it work if we rename the file to `crowdin.json`? This will reduce some cognitive dissonance for me in the future.

Reply:
I'll do that - I'll just need to update the CrowdIn GH action configuration to point to that new filename (since by default it only uses `crowdin.yml`.